wallet-new/src/Cardano/Wallet/API/V1/Transactions.hs

```diff
@@ -9,7 +9,7 @@ import           Servant
 
 type API
     =    "transactions" :> Summary "Generates a new transaction from the source to one or multiple target addresses."
-                        :> ReqBody '[JSON] Payment
+                        :> ReqBody '[JSON] NewPayment
```

Comment:
`New Payment`, for an easy refactoring. You might need to add an extra entry to the type family in `V1.Types`.

EDIT: Actually, let's use `Payment` directly. There is no need for a `newtype` here.

The type families `New` and `Update` are useful when we have a somewhat standard CRUD resource we can update, but for a `Transaction` doesn't make sense. If we want to be consistent with all we have done so far, maybe we should have a `New Transaction` which maps to a `Payment`. Thoughts? cc @akegalj @et4te @Martoon-00

Reply:
Hm, Interesting. I guess it would make code more readable.